include/onnxruntime/core/framework/op_kernel.h

```diff
@@ -83,15 +81,14 @@ class OpKernelContext {
     try {
       return p_ml_value ? &(p_ml_value->Get<T>()) : nullptr;
     } catch (const std::exception& /*e*/) {
-      throw OnnxRuntimeException(ORT_WHERE_WITH_STACK, "Missing Input: " + kernel_->Node().InputDefs()[index]->Name());
+      throw OnnxRuntimeException(ORT_WHERE_WITH_STACK, "Missing Input: " + kernel_.Node().InputDefs()[index]->Name());
     }
   }
 
   // Fetch output (non-tensor) with specified index.
   template <typename T>
   T* Output(int index) {
-    if (index < 0 || index >= OutputCount())
-      return nullptr;
+    if (index < 0 || index >= OutputCount()) return nullptr;
 
```

Comment:
Is it possible to separate out all the formatting changes into a separate PR? 

clang-format doesn't automatically make these changes for me when using the checked in .clang-format file so they seem optional and add a lot of noise to the PR.

Reply:
Yes, I'll try